benchmarks/benchmark_throughput.py

```diff
@@ -78,6 +78,7 @@ def run_vllm(
     enable_prefix_caching: bool,
     enable_chunked_prefill: bool,
     max_num_batched_tokens: int,
+    distributed_executor_backend: str,
```

Comment:
Should be `Optional[str]` since you set a default of `None` in the CLI arguments. Perhaps it would be simpler to directly set `default="ray"`?

Reply:
I figured leaving it as None would let `LLM` use whatever's default.

Reply:
In that case, just need to update the type annotation.